src/fty-sensor-gpio.conf

```diff
@@ -0,0 +1,2 @@
+# create runtime directories for fty-sensor-gpio
```

Comment:
Note that `/usr` (perhaps coming from the OS image archive) can quite hold read-only data, such as template rulesets to instantiate actual copies of the rules under `/var`, etc.